docs/generated/checks.md

```diff
@@ -482,6 +482,21 @@ dirs:
 port: 22
 protocol: TCP
 ```
+## target-port-specification
```

Comment:
I would name this something like `invalid-target-ports`

Reply:
✅